File: 1848-sum-of-unique-elements/sum-of-unique-elements.cpp
class Solution {
public:
    int sumOfUnique(vector<int>& nums) {

        int freq[101] = {0};
        int sum = 0;

        for (int num : nums) {
            freq[num]++;
        }

        for (int i = 0; i < 101; i++) {
            if (freq[i] == 1) {
                sum += i;
            }
        }

        return sum;
    }
};

File: 1848-sum-of-unique-elements/sum-of-unique-elements.java
class Solution {
    public int sumOfUnique(int[] nums) {
        Arrays.sort(nums);

        int ans = 0;
        int i = 0;

        while (i < nums.length) {
            int j = i;

            while (j < nums.length && nums[j] == nums[i]) {
                j++;
            }

            if (j - i == 1) {
                ans += nums[i];
            }

            i = j;
        }

        return ans;
    }
}